orders/views.py

```diff
@@ -54,3 +55,19 @@ def get(self, request):
         }
 
         return JsonResponse({"results": results}, status=200)
+
+    @login_decorator
+    def delete(self, request):
+        try:
+            data = json.loads(request.body)
+
+            OrderItem.objects.filter(
+                order__status_id=OrderStatus.Status.ON_CART.value,
+                product_id=data["product_id"],
+            ).delete()
+
+            return JsonResponse(
+                {"message": f"PRODUCT {data['product_id']} DELETED"}, status=200
+            )
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
```

Comment:
REST API에서 DELETE를 구현할 때에는 path parameter를 사용해서 구현해야 합니다!
`DELETE /cart/1` 1번 카트를 지워라!

```suggestion
    @login_decorator
    def delete(self, request, cart_id):
        try:
            OrderItem.objects.get(product_id=product_id, ..).delete()

            return JsonResponse(
                {"message": f"PRODUCT {data['product_id']} DELETED"}, status=200
            )
        except KeyError:
            return JsonResponse({"message": "KEY_ERROR"}, status=400)
```